src/zkevm_specs/evm/execution/error_insufficient_balance.py

```diff
@@ -0,0 +1,40 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag, AccountFieldTag
+from ..opcode import Opcode
+
+
+def insufficient_balance(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    # TODO: add Create / Create2 in the future
+    instruction.constrain_in(opcode, [FQ(Opcode.CALL), FQ(Opcode.CALLCODE)])
+    # TODO: for create/create2 have different stack from Call, will handle it in the future
+    # Lookup values from stack
+    instruction.stack_pop()
+    instruction.stack_pop()
+    value_rlc = instruction.stack_pop()
+    instruction.stack_pop()
+    instruction.stack_pop()
+    instruction.stack_pop()
+    instruction.stack_pop()
+    is_success_rlc = instruction.stack_push()
+    # if is_success_rlc value is zero then decode RLC should also be zero
+    instruction.constrain_zero(is_success_rlc)
+
+    value = instruction.rlc_to_fq(value_rlc, 31)
```

Comment:
Instead of `31`, please use https://github.com/privacy-scaling-explorations/zkevm-specs/blob/4ed0916f8dea328775fcb6e903b12db0551be8ac/src/zkevm_specs/util/param.py#L2